AK/Concepts.h

```diff
@@ -79,6 +79,16 @@ concept ArrayLike = requires(ArrayT array, SizeT index)
     -> SameAs<RemoveReference<ContainedT>*>;
```

Comment:
Typo in commit message: `dearly` -> `clearly`

Reply:
No, that's a normal English word :^)

Reply:
Oh, my bad! Learned something new then :^)